### ISSUE TITLE
Bug or incorrect usage?

### ISSUE THREAD

**Issue by tudorels** [vitmalina/w2ui#1378]
Hello,
I have a page with 2 grids. The first grid show some "master data" and when I click row I want to add some details in the second grid. Well its very simple, but I have a problem when I used the code(onClick event for grid1):
 ```
grid2.add({ recid: recid, cat: cat, s: s, date_in: data_in, data_out: data_out, cost: cost, observation: observation} );
grid2.refresh(); 
```
the data are not showing in the second grid, but records are counted (see the picture).
If I click on a column to do sorting (or do any search or click reload button on grid) the records appears in the grid. I use w2ui v 1.5 RC1.
Can somebody help or I found a bug? Thank you!

![2grids](https://cloud.githubusercontent.com/assets/15701091/20132905/d26d43d0-a66d-11e6-9e8c-a523375e55e2.png)

**Comment by mrg2001** [comment]
Please create a JS fiddle to illustrate the problem. In demos it works, here are demos for 1.4 
http://w2ui.com/web/demos/#!grid/grid-18
http://w2ui.com/web/demos/#!grid/grid-19

It works in my local for 1.5 too.

**Comment by tudorels** [comment]
Now I can't reproduce in jsfiddle, but I was able to run a separate example and is working using your example grid-19. It's possible that the issue exist because I'm using kickstart in my app?

**Comment by mrg2001** [comment]
It is highly unlikely kickstart is the problem. I sounds to me like you found the problem. I am closing the issue.

**Comment by tudorels** [comment]
I don't understand what are you saying, for sure I found a problem, but the question is what are causing this problem. I will try to fiddle this.

**Comment by mrg2001** [comment]
Can you please post your code here?

**Comment by mpf82** [comment]
@mrg2001 tudorels wrote here (https://github.com/vitmalina/w2ui/issues/1381) that the issue is with jQuery 3.x

I've created a fiddle based on  http://w2ui.com/web/demos/#!grid/grid-19 with jQuery 3.x to demonstrate the issue: http://jsfiddle.net/dv78xLvw/

the same fiddle with jQuery 2.2.4 http://jsfiddle.net/dv78xLvw/1/

**Comment by mrg2001** [comment]
I found the issue. In jQuery 2.x when I do $('#no-element').height() it returns null if there are no elements like this. But in 3.x it returns undefined. However if this is used in a math expression
```js
null / 0 - is zero
undefined / 0 is NaN
```
This is why it was breaking. I have fixed it in both branches 1.5.x and 1.4.x

**Comment by tudorels** [comment]
Thank you! Glad to see it's fixed!
